shared/src/commonMain/kotlin/ui/composables/credentials/EuPidCredentialAgeDataCard.kt

```diff
@@ -42,6 +42,7 @@ fun EuPidCredentialAgeDataCardContent(
 ) {
     val agesAtLeastN = listOf(
         18 to credentialAdapter.ageAtLeast18,
+        65 to credentialAdapter.ageAtLeast65,
```

Comment:
why only these two?

Reply:
That's only the "overview" card, showing a few selected attributes